tests/test_util.py

```diff
@@ -15,6 +15,7 @@
     pytest.importorskip("tensorflow")
 import tensorflow as tf
 
+import jax.numpy as jnp
```

Comment:
maybe make some guard against jax not installed...?

Reply:
but we have `jax` in `requirements_dev.txt`?..

Reply:
yeah... i guess we will fix it when we shard stuff

Reply:
..and I'd want to know in what env's it would fail :)

Reply:
in the end, it was good call on your side, as always :)
(it fails on Win as our test suite fails to build the required jaxlib)
fixed.